src/franzininho-element.stories.ts

```diff
@@ -0,0 +1,20 @@
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { action } from '@storybook/addon-actions';
+import './franzininho-element';
+
+storiesOf('Franzininho', module)
```

Comment:
Can you please change the story to the new storybook 6 format? 

You can find an example in the flame-sensor or slide-potentiometer element stories